### ISSUE TITLE
Question about the integrations evaluated with and without periodic boundary condition

### ISSUE THREAD

**Issue by yanghan234** [pyscf/pyscf#1605]
Dear developers,

To my understanding, the implementation of periodic boundary condition in PySCF first evaluates overlap matrices and other 1e, 2e integrals on a real-space grid and then transforms them to a k-point grid. (Correct me if this is not the case) This strategy is very similar to the CRYSTAL code. For example, the overlap matrices write,
$$S_{\mu\nu}(\mathbf{k}) = \sum_{\mathbf{R}}\exp(i\mathbf{k}\cdot\mathbf{R}) S_{\mu\nu}(\mathbf{R})$$, where $\mathbf{k}$ and $\mathbf{R}$ are k-point and real-space lattice grids, respectively. It seems to me that, given 1x1x1 k-point sampling, the overlap evaluated at gamma point $\mathbf{k}=\Gamma=0$ would be completely equivalent to the real-space integral,
$$S_{\mu\nu}(\mathbf{k}=0)=S_{\mu\nu}(\mathbf{R}=0) = \int\phi_\mu(\mathbf{r})\phi_\nu(\mathbf{r})\mathrm{d}\mathbf{r}$$

However, by running a minimal test case below, I find that they are not equal, 
```python
from pyscf import gto
from pyscf.pbc import gto as pbcgto
import numpy as np

cell = np.array([ [0.0000, 1.7800, 1.7800],
                  [1.7800, 0.0000, 1.7800],
                  [1.7800, 1.7800, 0.0000] ])

mol = gto.M(atom="H 0 0 0; H 0.8900 0.8900 0.8900", basis="sto-3g")
pbcmol = pbcgto.M(atom="H 0 0 0; H 0.8900 0.8900 0.8900", basis="sto-3g", a=cell)

kpts = pbcgto.make_kpts(pbcmol, [1,1,1])

ovlp = mol.intor("int1e_ovlp")
pbcovlp = pbcmol.pbc_intor("int1e_ovlp", kpts=kpts)
print(f"{ovlp=}")
print(f"{pbcovlp=}")
```
Here are the outputs:
```bash
ovlp=array([[1.        , 0.24188075],
       [0.24188075, 1.        ]])
pbcovlp=[array([[1.62447222, 1.24531681],
       [1.24531681, 1.62447222]])]
```
I'm not very faimiliar with the periodic GTO calculaitons, so I must have made a mistake in writing the code or interpreting the method implemented in PySCF. Could you please help me with this issue. Thank you!

**Comment by fishjojo** [comment]
The basis functions for periodic calculations are crystalline Gaussian orbitals, which have the form
$\phi_k (r) = \sum_R e^{ik \cdot R} \phi (r-R)$
Your first equation is correct, so even at the Gamma point, there is a summation over the periodic images, which differs from the molecular case unless you use a supercell.
You can refer to these two papers:
10.1021/acs.jctc.7b00049
10.1063/1.4998644

**Comment by tberkel** [comment]
To add to @fishjojo's response ... the first equality in your final equation is not correct, and the corrected version might help with your confusion:
$$S_{\mu\nu}(\mathbf{k}=0)=\sum_{\mathbf{R}} S_{\mu\nu}(\mathbf{R})=\sum_{\mathbf{R}} \int \phi_\mu(\mathbf{r})\phi_\nu(\mathbf{r}+\mathbf{R}) d\mathbf{r}$$

**Comment by yanghan234** [comment]
Thanks! I do understand that 
$$S_{\mu\nu}(\mathbf{k}=0) = \sum_\mathbf{R}S_{\mu\nu}(\mathbf{R})$$

But I did not see the parameter to set this $R$ grid in the input. So, I mistakenly assumed that the summation was reduced to only $\mathbf{R}=0$. 
Then, my question is what is the default $\mathbf{R}$ grid in the code? The `rcut` parameter of the `Cell` class?

**Comment by fishjojo** [comment]
Yes, `cell.rcut` determines the lattice summation range, see https://github.com/pyscf/pyscf/blob/8b3fef8cf18f10d430261d4a8bea21fadf19bb1f/pyscf/pbc/tools/pbc.py#L482
And `cell.rcut`, if not set by users, is determined by the integral precision `cell.precision`.
